src/main/java/exam2/lotto/PurchaseLottoList.java

```diff
@@ -0,0 +1,117 @@
+package exam2.lotto;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+//로또 구입 번호 클래스
+public class PurchaseLottoList {
+
+    private final List<List<Integer>> numberList = new ArrayList<>();
+    private final static int lottoStartNumber = 1;
+    private final static int lottoEndNumber = 45;
```

Comment:
매직변수 추출 좋습니다.  👍 

로또번호의 범위가 변경될때 메서드까지 안내려다 보고 상위에서 제어하면 되겠군요!
상수의 컨벤션은 대문자 SNAKE UPPER형식이 컨벤션이에요 피드백 수정하시다가 놓치신거 같아요 ㅠㅠ

```suggestion
private final static int LOTTO_START_NUMBER = 1;
private final static int LOTTO_END_NUMBER = 45;
```